documentation/modules/ref-address-example-message-redelivery.adoc

```diff
@@ -24,4 +24,6 @@ message redelivery settings that are in force.
 
 If the address has a `deadletter` address assigned, messages that have exceeded the maximum number of delivery attempts are automatically moved to it.
 
-If the address has no `deadletter` address assigned, the system drops the messages. These messages are lost permanently.
+If the address has no `deadletter` address assigned, the system drops the messages. These messages are lost permanently. footnote:[The standard address
```

Comment:
@hmaclean73 FYI

Reply:
Not sure that this should be a footnote. Perhaps we could remove the footnote and start the text directly after the last sentence with "However, the standard address space supports..."

Reply:
Actioned.